src/task.ts

```diff
@@ -90,9 +92,11 @@ export default class Task extends EventTarget {
           taskTime = this.bench.now() - taskStart;
         }
 
-        samples.push(taskTime);
         this.runs += 1;
         totalTime += taskTime;
+        samples.push(taskTime);
+        if (taskTime > max) max = taskTime;
+        if (taskTime < min) min = taskTime;
```

Comment:
what do these lines do exactly?

Reply:
Get the max and min values, This way we don't have to call `sort()` again.